src/nav/ModalNavBar.js

```diff
@@ -24,17 +24,14 @@ export default function ModalNavBar(props: Props) {
   const insets = useSafeAreaInsets();
 
   return (
-    <>
-      <View style={{ height: insets.top }} />
+    <View style={{ borderBottomWidth: 1, backgroundColor, paddingTop: insets.top }}>
       <View
         style={[
           {
             borderColor: 'hsla(0, 0%, 50%, 0.25)',
             flexDirection: 'row',
             height: NAVBAR_SIZE,
             alignItems: 'center',
-            borderBottomWidth: 1,
-            backgroundColor,
```

Comment:
Hmm, is this quite right? The `borderColor` is getting separated from the `borderBottomWidth`.

For that matter: is there a reason not to collapse these into a single wrapper View? I don't think I'm spotting a way that we're using the distinction.

Reply:
I guess the next commit turns the outer one into a `SafeAreaView`. But that's supposed to be just like a `View` with one extra feature, so I think the same question applies.

Reply:
> is there a reason not to collapse these into a single wrapper View? I don't think I'm spotting a way that we're using the distinction.

The existing `View` sets an explicit height of `NAVBAR_SIZE` (58), so the part that holds the content gets shrunk. Though I guess we could get rid of that explicit height; we recently removed one in #4442.

Reply:
I think the situation is similar with `ModalSearchNavBar`.

Reply:
Ah I see, because the padding comes out of the `height`.

We could also handle that directly, I think, by adding the desired padding to the height when combining these two `View`s:
```js
  paddingTop: insets.top,
  height: insets.top + NAVBAR_SIZE,
```

But removing NAVBAR_SIZE sounds good in any case.

Reply:
> We could also handle that directly, I think, by adding the desired padding to the height when combining these two `View`s:
> 
> ```js
>   paddingTop: insets.top,
>   height: insets.top + NAVBAR_SIZE,
> ```

Oh, but this gets stickier when converting to `SafeAreaView`, doesn't it.

Anyway, happy to remove `NAVBAR_SIZE` :slightly_smiling_face: